github/ci/prow-deploy/files/orgs.yaml

```diff
@@ -714,6 +715,11 @@ orgs:
           - isaacdorfman
         repos:
           terraform-provider-kubevirt: admin
+      rerun-jobs-hco:
+        description: can rerun failed job in the hyperconverged-cluster-operator repository
+        maintainers:
```

Comment:
nit: I even think that we could also just use members: here instead, but it should be fine. Just remember that while you can change the members of the team then again peribolos would overwrite your changes.